package.json

```diff
@@ -8,9 +8,9 @@
   },
   "scripts": {
     "compile": "truffle compile",
-    "compile-all": "truffle compile --all",
+    "compile-all": "truffle compile --all --reset",
```

Comment:
`--reset` option is not available with `truffle compile` command.
It can only be used with [truffle migrate](https://truffleframework.com/docs/truffle/reference/truffle-commands#migrate)  command.

Reply:
I've been using it locally always, as it's resetting some cache, which makes the contracts to rebuilt all the time (even if there are not changes). However, I would remove it from travis as it's always a fresh build.

Reply:
`truffle compile --all` is sufficient to compile all the contracts again. Truffle simply ignores what follows after `--all` in this case.